### PR TITLE
add header

### PR DESCRIPTION
**Pull request by mim24sd**
In this pull request we want to :
1. Make our header component.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import { render } from "react-dom";
+import Home from "./pages/Home";
 
 const App = () => {
-  return <div></div>;
+  return (
+    <div>
```
Comment (blacksrc) [inline src/App.jsx]: Why we have `div` here? Maybe you could remove it 🤔

File: src/pages/Albums/index.jsx [hunk 2]
```diff
@@ -1,5 +1,11 @@
+import Header from "../../components/Header";
+
 const Albums = () => {
-  return <div></div>;
+  return (
+    <div>
```
Comment (blacksrc) [inline src/pages/Albums/index.jsx]: Maybe you could remove this?

File: src/pages/Home/index.jsx [hunk 3]
```diff
@@ -1,5 +1,11 @@
+import Header from "../../components/Header";
+
 const Home = () => {
-  return <div></div>;
+  return (
+    <div>
```
Comment (blacksrc) [inline src/pages/Home/index.jsx]: Can you remove the `div`?

File: src/pages/PlayLists/index.jsx [hunk 4]
```diff
@@ -1,5 +1,11 @@
+import Header from "../../components/Header";
+
 const PlayLists = () => {
-  return <div></div>;
+  return (
+    <div>
```
Comment (blacksrc) [inline src/pages/PlayLists/index.jsx]: Can you remove the `div`?

File: src/pages/Singles/index.jsx [hunk 5]
```diff
@@ -1,5 +1,11 @@
+import Header from "../../components/Header";
+
 const Singles = () => {
-  return <div></div>;
+  return (
+    <div>
```
Comment (blacksrc) [inline src/pages/Singles/index.jsx]: Can you remove the `div`?

**Comment by blacksrc** [comment]
Please fix the deployment on netlify. So we can have a live demo 🎉

File: src/App.jsx (hunk 1 above)
Comment (mim24sd) [reply src/App.jsx]: I added this for future but you are right.That was wrong.

**Review by blacksrc — APPROVED** [review]

**Pull request merged**